### PR TITLE
Pytest travis

### PR DESCRIPTION
**Pull request by gemabarni**
This pr adds continuous integration with travis and testing capability with pytest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,4 +17,13 @@ Python data structures.
 $ pip install cbor-decoder
 ```
 
+## Run tests
+
+```
+$ pip install -U pytest
```
Comment (gerhardberger) [inline README.md]: I think we should be using `pip3` here. At least that what succeeded for me installing.

File: README.md [hunk 2]
```diff
@@ -15,4 +17,13 @@ Python data structures.
...
+$ pytest
```
Comment (gerhardberger) [inline README.md]: After installing there was no `pytest` command for me. Is there some other way to install or run the tests?

File: README.md [hunk 3]
```diff
@@ -15,4 +17,13 @@ Python data structures.
...
+```
+
+Please use the naming convention `test_*.py` for test files, as this is the required format for pytest.
```
Comment (gerhardberger) [inline README.md]: We can omit the 'Please' here. 😄

**Review by gerhardberger — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gemabarni) [reply README.md]: It is more of a convention. `pip` always uses the default python version. Probably it is python2 in your case. Replaced it with `pip3` though, just to make sure everyone uses python3.

File: README.md (hunk 2 above)
Comment (gemabarni) [reply README.md]: Fixed it, now it runs with `python scripts/test.py` as well.

File: README.md (hunk 3 above)
Comment (gemabarni) [reply README.md]: I was just trying to be a nice guy 😢

**Review by gerhardberger — APPROVED** [review]

**Pull request merged**